src/Task/Progpilot.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * Progpilot task
+ */
+class Progpilot extends AbstractExternalTask
+{
+    /**
+     * @return string
+     */
+    public function getName()
+    {
+        return 'progpilot';
+    }
+
+    /**
+     * @return OptionsResolver
+     */
+    public function getConfigurableOptions()
+    {
+        $resolver = new OptionsResolver();
+        $resolver->setDefaults(
+            [
+                'config_file'  => '.progpilot/configuration.yml',
```

Comment:
Isn't it better to set this value to null so that the defaults from the cli tool are being used?

Reply:
Defaults values from the tool are also used when the configuration file specified in the cli doesn't exist so if someone doesn't have a file *.progpilot/configuration.yml* the defaults values from the tool will be used and if someone has a file *.progpilot/configuration.yml* but has forgotten to define it in the parameters of the task the values of *.progpilot/configuration.yml* will be used.